Documentation/Signoffs/IntercepterControllerSubSystem.md

```diff
@@ -0,0 +1,78 @@
+
+Figure 2: Buildable Schematic
+
+## **Analysis:**
+
+Stepper Motor Integration:
+
+The stepper motor driver converts a pulse signal into a precise position and can also make the rotation of the motor bidirectional. With this function and the orientation of the motors we are able to create an x and y axis for the firing mechanism to move on. This allows 30 set coordinates to be programmed for possible incoming attacks. This component was pick by the Mechanical Engineering portion of our team, however, it is important to this subsystem because of its direct integration within the Arduino. Through GPIO communication, as stated above, the Arduino UNO R3 commands the stepper motor driver to adjust the interceptor's position, aligning it with the golf ball's predicted path. Real-time trajectory data calculated by the processor is sent to the Arduino that will guides precise motor movements, ensuring accurate interception. The stepper motor will be put into a zero position. While it is in its zero position, the processor can count how many steps the motor takes to determine the correct aiming position. This can apply for the X-Axis and Y-Axis of the system to be able to reset the interceptor back to the zero position after each time it aims and fires at the incoming golf ball [6].
+
+Arduino Inergration:
+
+As stated before the stepper motor drive given was picked by the ME team. This component was picked through testing and consideration of use by them. The Arduino UNO R3 seamlessly integrates with the TB6600 Stepper Motor Driver to control the interceptor's aiming mechanism [7]. With that being said an Arduino was picked to be used within this subsystem to control the driver. Through research it is found to be the most commonly used device to pair with the stepper motor driver and with a large data base of research, experiments, and recommendations. This also allows the team to alleviate some of the coding within the processor making it easier to trouble shoot problems, breaks up coding assignments, and allows for specific testing with the ME team to fine tune the speed and positions.The arduino will need to set the motors to move at 200 RPM [9] to allow ample time for firing. The worse possible case, with it needing to fire at the first or last line, it can take anywhere between 340 and 577 depending on the diameter of the position mechanism. The motor used for the firing mechanism runs at 10,000 RMPs and will be plenty fast enough to hit the object in time. The arduino will also communicate with the The Jetson Nano which will communicate with the Arduino using SPI interface. The pins: 'spi_ce0_n' (Enable), 'spi_clk' (Clock), 'spi_mosi' (Master Out Slave In), and 'spi_miso' (Master In Slave Out) the pins will handle data transmission from the Jetson Nano to the Arduino and vice versa. By connecting these pins it will be able to send location information and firing information while receiving data from the Arduino.
```

Comment:
"between 340 and 577" where are the units?

How do you know that 200RPM will allow ample time for firing?